### DISCUSSION TITLE
Feature Request: Create Portal Team Member with NO ACCESS to backoffice

### DISCUSSION THREAD

**Discussion by c9mb** [umbraco/Umbraco.Cloud.Issues#791] [Ideas]
### Issue description

I suspect it's not a common-place requirement, but there are times when I'd like to be able to give someone access to the portal team for the project, but leave them with NO ACCESS to the backoffice by default.

The reason for this is quite simple - there is no way to remove a backoffice account once a user has logged in - and having someone log in when it wasn't really necessary means an added management task to check backoffice user accounts and disable them.

I note that from the portal it's possible to create a user with access to backoffice but NO ACCESS to the team, so what I'm asking for is the option to be able to reverse that, and create a user with access to the team but NO ACCESS to the backoffice.

A classic case-in-point here is when support need access to the portal team of a project - they don't normally need access to the backoffice (or will specifically ask if they do) - and being able to prevent an unnecessary backoffice  log-on, can avoid leaving me with a backoffice account I can't remove - and then needing to explain the reason for them to the site owner.

**Comment by sajumb** [comment]
Thanks for the feature request. We understand the need for this and we are likely to consider this in the next sprint that starts on Monday.
